DBTableToHBaseTable/README.rst

```diff
@@ -0,0 +1,57 @@
+Batch Database Table To CDAP HBase Table Adapter Configuration
+==============================================================
+
+The ETL Batch Template can be used to create an Adapter that reads from a Batch Source and persists it to a Sink.
+In this example, we will read an entire DB table in batch and use a TableSink to write the database table's rows to HBase.
+
+The config.json contains a sample Adapter configuration that you can use to accomplish the above task. Our sample Adapter uses the following components:
+
+- etlBatch Application Template, since we want to perform ETL in batch
+- Database source, to read data from the database table 
+- Table sink, to write the Kafka events to a Stream
+
+You can create and start the Adapter by using the CDAP CLI (or you can use the UI for a more visual approach).
+
+Note: You need to fill in the following configurations in config.json before creating the Adapter.
+
+Configurations for the Database Table Source
+  1. ``driverClass`` - This is the JDBC driver class to use to connect to your database.
+  2. ``connectionString`` - This is the JDBC conection string including the database name.
+  3. ``tableName`` - This is the table you wish to import from.
```

Comment:
Avoid ending the sentence with a preposition: change 'you wish to import from' to 'from which you wish to import'